org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -1343,13 +1343,44 @@ protected void load (int ID, String trxName)
 		}
 	}	//	load
 
+	/**
+	 * Load record with UUID
+	 * 
+	 * @param uuID    UUID
+	 * @param trxName transaction name
+	 */
+	public void loadByUU(String uuID, String trxName)
+	{
+		// reset new values
+		m_newValues = new Object[get_ColumnCount()];
+		checkImmutable();
+
+		if (log.isLoggable(Level.FINEST))
+			log.finest("uuID=" + uuID);
+		if (Util.isEmpty(uuID, true))
```

Comment:
sorry, I means this should be at top, i.e comes before the "//reset new values" line.

Reply:
Done